Modules/Numerics/Optimizers/include/itkLBFGSBOptimizer.h

```diff
@@ -194,6 +194,13 @@ class ITKOptimizers_EXPORT LBFGSBOptimizer : public SingleValuedNonLinearVnlOpti
   const std::string
   GetStopConditionDescription() const override;
 
+  /** Returns false unconditionally because LBFGSBOptimizer does not support using scales. */
+  bool
+  GetCanUseScales() const override
```

Comment:
Thanks Hans! BTW Why do you call it  **Get**CanUseScales()?  Why not just `CanUseScales()`?

Reply:
Thanks for the input. Changed based on your suggestion.